src/services/token.js

```diff
@@ -48,6 +49,21 @@ class TokenService {
       expiresIn: `${this.expirationInDays} days`,
     });
   }
+
+  /**
+   * @param {import('express').Request} request
+   * @param {import('express').Response} response
+   */
+  // eslint-disable-next-line class-methods-use-this
+  deleteToken(request, response) {
```

Comment:
Services should not be aware of the HTTP/express context. So it's a bad practice to manipulate these variables in a service that was not depending on express before.

You should refactor this to extract calls to request and response and let them in the route, and do the rest in the service.